lib/bolt/module_installer.rb

```diff
@@ -216,27 +184,5 @@ def install_puppetfile(path, moduledir, config = {})
 
       ok
     end
-
-    # Asserts that an existing Puppetfile is managed by Bolt.
-    #
-    private def assert_managed_puppetfile(puppetfile, path)
```

Comment:
I like that this logic is moved to the Puppetfile object itself, so this class is really just an interface between the CLI and the Puppetfile